mobile/lib/pages/common/headers_settings.page.dart

```diff
@@ -60,7 +60,7 @@ class HeaderSettingsPage extends HookConsumerWidget {
 
     return Scaffold(
       appBar: AppBar(
-        title: const Text('advanced_settings_proxy_headers_title').tr(),
+        title: const Text('headers_settings_tile_title').tr(),
```

Comment:
We can go one step further and replace these hardcoded strings with the same key names from IntlKeys class. So something like this


```suggestion
        title: const Text(IntlKeys.headers_settings_tile_title).tr(),
```

Reply:
@shenlong-tanwen there are 90 instances of such `Text('').tr()` in the source code, should this perhaps be a separate PR or do you want me to just fix it in the files touched by this PR?

Reply:
Just the ones modified in this PR is fine. Only very few keys across the codebase use the generated class now. But it'd be great to start migrating the existing ones to use the class